test/port-finder.test.js

```diff
@@ -152,32 +153,43 @@ describe('with no existing servers', function () {
   });
 });
 
-describe.each([
-  ['getPort()', false, portfinder.getPort],
-  ['getPort()', true, portfinder.getPort],
-  ['getPortPromise()', true, portfinder.getPortPromise],
-])(`the %s method (promise: %p)`, function (name, isPromise, method) {
-  test('with startPort less than or equal to 80', function (done) {
-    if (isPromise) {
-      method({ startPort: 80 })
```

Comment:
so we had this test which was hiding the fact that startPort didn't even work - we thought we set it to port 80 and then tested it went higher but it was the basePort that was making it do that - now its correct